pkg/ccl/importccl/read_import_mysql.go

```diff
@@ -598,13 +598,16 @@ func mysqlColToCockroach(
 	}
 
 	if col.Default != nil && !bytes.EqualFold(col.Default.Val, []byte("null")) {
-		expr, err := parser.ParseExpr(string(col.Default.Val))
+		exprString := string(col.Default.Val)
+		if col.Default.Type == mysql.StrVal {
+			exprString = fmt.Sprintf("'%s'", col.Default.Val)
```

Comment:
could you add a test for a default containing a single quote to make sure escaping happens properly?

Reply:
Done. Decided I'd rather skip the parser than try to quote/escape just to make it though the parser.